drivers/hyperv/powershell.go

```diff
@@ -61,7 +61,7 @@ func hypervAvailable() error {
 	}
 
 	resp := parseLines(stdout)
-	if resp[0] != "Hyper-V" {
+	if resp == nil || resp[0] != "Hyper-V" {
```

Comment:
Perhaps check for `len(resp) = 0` as well?

Reply:
good call, done.